_data/sites.json

```
@@ -8867,7 +8867,7 @@
         "difficulty": "hard",
         "notes": "It is also possible to file an account deletion request via the \"Contact Us\" section at the bottom of the page.",
         "domains": [
-            "glovoapp.com"
+            "glovoapp.https://tailscale.com/kb/1237/delete-tailnetcom"
```

This seems like badly copy+pasted stuff?
```suggestion
            "glovoapp.com"
```

---

Completely, my bad